### PR TITLE
Stats: Updates to PromoCard component.

### PR DESCRIPTION
**Pull request by a8ck3n**
#### Proposed Changes

Addressing the feedback in #69819.

1. Use CSS variables for colours.
2. Add working links to title/message text.
3. Localize said text.
4. Accept a className property for easier customization.
5. Clean up props to accommodate above changes.
6. Update SVG data so we get the nice white separation between the icons.

Updated snap:

PromoSnap2

#### Testing Instructions

* Check out this branch locally.
* Spin up Storybook via `yarn workspace @automattic/components run storybook`.
* Check the PromoCard stories. Ensure they behave as expected.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to #69230

### CODE REVIEW

File: packages/components/src/promo-card/index.tsx [hunk 1]
```diff
@@ -1,29 +1,55 @@
...
+				'Check your stats on-the-go and get real-time notifications with the Jetpack mobile app.'
+			);
+		} else {
+			string = translate(
+				'Visit {{a}}wp.com/app{{/a}} or scan the QR code to download the Jetpack mobile app.',
+				{ components }
+			);
+		}
+		return string;
+	};
```
Comment (jsnmoon) [inline packages/components/src/promo-card/index.tsx:36]: Could we inline this within the component tree? Unless we're planning on using these values multiple times in the tree, there's no need to declare them as separate variables outside of the returned tree.

**Review by jsnmoon — APPROVED** [review]
🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7623422

Thank you @a8ck3n for including a screenshot in the description! This is really helpful for our translators.

File: packages/components/src/promo-card/index.tsx (hunk 1 above)
Comment (a8ck3n) [reply packages/components/src/promo-card/index.tsx:36]: Addressed in #70023.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7623422).

**Pull request merged**